server/app/api/v1/chat.py

```
@@ -26,13 +26,12 @@
 
 @router.post("/text/text")
 async def chat_text_to_text(
-    request_body: TextChatRequest,
+    agent_id: str = Form(...),
+    message: str = Form(...),
+    image_file: Optional[UploadFile] = File(None),
```

Ruff (0.8.2)

Do not perform function call `File` in argument defaults; instead, perform the call within the function, or read the default from a module-level singleton variable

(B008)